sunpy/timeseries/tests/test_timeseries_factory.py

```diff
@@ -150,6 +150,21 @@ def test_implicit_new_goes16(self):
         ts_goes = sunpy.timeseries.TimeSeries(new_goes16_filepath)
         assert isinstance(ts_goes, sunpy.timeseries.sources.goes.XRSTimeSeries)
 
+    def test_implicit_goes_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(goes_filepath)
+        assert ts_goes.satellite_number == 15
+
+    def test_implicit_new_goes15_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(new_goes15_filepath)
+        assert ts_goes.satellite_number == 15
+
+    def test_implicit_new_goes16_satno(self):
+        # Test a GOES TimeSeries for satellite number
+        ts_goes = sunpy.timeseries.TimeSeries(new_goes16_filepath)
```

Comment:
Since this is actually a goes 17 file, I wonder if we should rename the variables while we are here?